modules/end-user-guide/pages/git-credential-store.adoc

```diff
@@ -1,25 +1,48 @@
-:navtitle: Git credential store
+:navtitle: Creating a {orch-name} Secret for a Git credentials store
 :keywords: user-guide, configuring, user, secrets
 :page-aliases: 
 
-[id="mounting-a-git-credential-store-into-a-workspace-container_{context}"]
-= Mounting a Git credentials store into a workspace container
+[id="mounting-a-git-credential-store-into-workspace-containers_{context}"]
+= Creating a {orch-name} Secret for a Git credentials store
 
-This section describes how to mount Git credentials store as secret from the user's {orch-namespace} into the file in single-workspace or multiple-workspace containers of {prod-short}.
+This section describes how to create a Secret containing a Git credentials store in the user's {orch-namespace}. The Git credentials store specified in the Secret is mounted as a file in the {devworkspace}'s containers.
 
 .Prerequisites
 
 * A running instance of {prod-short}.
+* You have CLI or GUI access to the {platforms-name} cluster of your organization's {prod-short} instance.
 
 .Procedure
-. Prepare Git credential file in the link:https://git-scm.com/docs/git-credential-store#_storage_format[Storage format].
-. Encode content of the file to the base64 format.
-. Create a new {platforms-name} secret in the {platforms-name} {orch-namespace} where a {prod-short} workspace will be created.
+. Prepare a Git credential file in the link:https://git-scm.com/docs/git-credential-store#_storage_format[Storage format].
 
-* The labels of the secret that is about to be created must match the set of labels configured in `che.workspace.provision.secret.labels` property of {prod-short}. The default labels are:
+. Encode content of the file to the base64 format.
 
-* `app.kubernetes.io/part-of: che.eclipse.org` 
-* `app.kubernetes.io/component: workspace-secret`:
+. Create a new {platforms-name} secret in the user's {orch-namespace}.
 +
-include::example$snip_{project-context}-git-credential-file-as-kubernetes-secret-into-the-file.adoc[leveloffset=+1]
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: git-credentials-secret
+  labels:
+    controller.devfile.io/git-credential: 'true'                 <1>
+    controller.devfile.io/watch-secret: 'true'
+  annotations:
+    controller.devfile.io/mount-path: __<path_in_container>__        <2>
+data:
+  credentials: __<base64_content_of_the_git_credentials_store>__     <3>
+----
++
+<1> The `controller.devfile.io/git-credential` label marks the secret as containing Git credentials. All Git credential secrets in the same {orch-namespace} will be merged into a single secret that would be mounted to the {devworkspace} containers. All the original secrets are kept in-tact.
+<2> The path in the {devworkspace}'s containers at which the merged Git credentials secrets are to be mounted. For example, if set to `/tmp/git-credentials`, the merged Git credentials store is mounted at `/tmp/git-credentials/credentials`.
```

Comment:
Let's say that I'm a new user who is reading the docs about what I have to do now.
How do I determine this path or decide what this path must be for my secret?

Reply:
Maybe a suggested path could be `che.eclipse.org/mount-path: /home/theia/.git-credentials` since that is the mount path used by [che-server](https://github.com/eclipse-che/che-server/blob/0189b2866afc82add1098c9fc181e27044197f2a/infrastructures/infrastructure-factory/src/main/java/org/eclipse/che/api/factory/server/scm/kubernetes/KubernetesGitCredentialManager.java#L60), as well as che-theia, however ultimately the mount path can be anything since there is a git-config (which contains the path to the mounted git credentials file) configmap that is mounted to the devworkspace containers.

I will update this PR.

Reply:
On second thought, since the mount-path is [optional](https://github.com/devfile/devworkspace-operator/blob/41e8ccb8623b80f0829c605f372e468bdf9fdda0/pkg/provision/automount/templates.go#L129-L132), I've removed it from the example secret, and added a new note that the mount-path is optional.